src/commands/sources/query.ts

```diff
@@ -19,9 +19,9 @@ export class QuerySourcesCommand extends Command {
   static description = 'Query sources for terms';
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   static flags: flags.Input<any> = {
-    identifiers: flags.string({
+    distributions: flags.string({
       description:
-        'Identifiers of sources to query, comma-separated, e.g. "nta,rkdartists"',
+        'URIs of distributions to query, comma-separated, e.g. "https://www.wikidata.org/sparql,https://data.netwerkdigitaalerfgoed.nl/rkd/rkdartists/sparql"',
```

Comment:
Though we’re using ‘IRI’ internally, I decided to keep ‘URI’ for all public-facing bits, including the GraphQL schema and this here.